packages/satusehat/src/Adaptor.js

```diff
@@ -103,7 +115,7 @@ export function post(path, data, params = {}, callback = s => s) {
 
 /**
  * Make a PUT request to Satusehat
- * @example
+ * @example <caption>Make a PUT request to update an Organization</caption>
  * put(
  *   "Organization/123",
  *  { "resourceType": "Organization", "active": false,
```

Comment:
Out of curiosity, if you're updating an existing organisation, presumably you don't need to send the resourceType? Would it be better or worse practice to exclude it?

Reply:
@josephjclark  yes it accepts the resource type in the body for a `PUT` request